src/ctree-app/features-section.html

```diff
@@ -0,0 +1,120 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/polymer/lib/dom-repeat.html">
+<link rel="import" href="../ctree-icons/ctree-landing-icons.html">
+
+<dom-module id="features-section">
+  <template>
+    <style>
+      :host {
+        display: block;
+        background-color: #FFFFFF;
+        color: #196848;
+        padding-top: var(--app-header-padding-fix);
+        margin-top: var(--app-header-margin-fix);
+      }
+      iron-icon {
+        --iron-icon-fill-color: #62A374;
+        --iron-icon-width: 80px;
+        --iron-icon-height: 80px;
+      }
+      .review-icon {
+        transform: rotate(40deg);
+      }
+      h1, h3 {
+        color: #62A374;
+      }
+      h1 {
+        text-align: center;
+        font-size: 2.9em;
+        letter-spacing: .02em;
+      }
+      h3 {        
+        text-transform: capitalize;
+        margin-top: 0;
+      }
+      p {
+        color: #222322;
+        font-size: 0.75em;
+        letter-spacing: .027em;
+        padding-top: 10px;
+      }
+      .container {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: space-evenly;
+        max-width: 1000px;
+        margin: auto;
+      }      
+      .collaboration, .search, .share, .review {
+        display: flex;
+        width: 350px;
+        margin-bottom: 50px;
+      }
+      .image {
+        margin-right: 50px;
+      }
+      /* Mobile Phones */
+      @media (max-width: 500px) {
+        h3 {
+          font-size: 1.25em;
+        }
+        div p {
+          font-size: 1.12em;
+          margin: 0 45px;
+        }
+        div .collaboration, div .search, div .share, div .review {
+          display: block;
+          width: 90%;
+        }
+        div .image {
+          margin-right: 0;
+        }
+      }
+    </style>
+
+    <h1>Features</h1>
+    <div class="container">
+      <dom-repeat items="[[sections]]" as="section">
+        <template>
+          <div class$="[[section.name]]">
+            <div class="image">
+              <iron-icon class$="[[section.name]]-icon" icon="[[section.name]]"></iron-icon>
```

Comment:
Similar concern here.  I'm wondering if you used the names as a way to get some consistent styling between the div and iron-icon, but I believe you can apply multiple classes for shared attributes.   So instead of naming a class for a type you can name it for a behavior/attribute (ex. "item wide" for the div and "wide" for the iron-icon).

Reply:
Actually, I was looking more at this during my testing and I saw the "-icon" added to the class name, but I don't see that class used anywhere, so we can probably just drop the class.  Also, see my comments about NOT using the section name for the icon name, and instead having a separate icon name and image name to set src (in case we want to provide an image path instead of a vector icon).

Reply:
The css style was on line 20:

```
.review-icon {
        transform: rotate(40deg);
      }
```
I will still take out the css style, just now the "review" icon looks like this:

![unrotated-review-icon](https://user-images.githubusercontent.com/44947175/70098640-79973900-15e1-11ea-9596-266ab97b2025.PNG)

> I'm wondering if you used the names as a way to get some consistent styling between the div and iron-icon, but I believe you can apply multiple classes for shared attributes.

And no. I implemented your request to name the div item, then erased the iron-icon's class$ attribute. Now this will be much cleaner as you intended.

Reply:
Yep, it's definitely not worth adding a special case just to update that icon.  It's better to have clean code and use a different icon if we want it to look differently.